# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/MacroAssembler.cpp
# Commit: 0c3169618f2f
# Full Hash: 0c3169618f2f8415ee2cc91816e2cc3a8af1693e
# Author: Steve Fink <sfink@mozilla.com>
# Date: 2021-09-09 15:22:41
# Regressor Bug: 1720422
# File Overlap Count: 2
# Description:
#   Bug 1720422 - Expose a templatized TypedArray<T> class in JSAPI r=jonco,nbp
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D119853
# ==============================================================================

diff -r 3005e92c10a3 -r 0c3169618f2f js/src/jit/MacroAssembler.cpp
--- a/js/src/jit/MacroAssembler.cpp	Wed Sep 08 23:46:51 2021 +0000
+++ b/js/src/jit/MacroAssembler.cpp	Wed Sep 08 23:46:56 2021 +0000
@@ -1845,7 +1845,8 @@
     branchPtr(Assembler::Equal, output, ImmPtr(constructor), &isTrue);
   };
 
-#define TYPED_ARRAY_CONSTRUCTOR_NATIVE(T, N) branchIsTypedArrayCtor(Scalar::N);
+#define TYPED_ARRAY_CONSTRUCTOR_NATIVE(_, T, N) \
+  branchIsTypedArrayCtor(Scalar::N);
   JS_FOR_EACH_TYPED_ARRAY(TYPED_ARRAY_CONSTRUCTOR_NATIVE)
 #undef TYPED_ARRAY_CONSTRUCTOR_NATIVE
 